python/ray/serve/_private/proxy.py

```diff
@@ -87,8 +87,8 @@
     "RAY_SERVE_HTTP_REQUEST_MAX_RETRIES cannot be negative."
 )
 
-TIMEOUT_ERROR_CODE = "timeout"
-DISCONNECT_ERROR_CODE = "disconnection"
+TIMEOUT_ERROR_CODE = "408"
+DISCONNECT_ERROR_CODE = "499"
```

Comment:
I couldn't find documentation online about what the status code is for client disconnect. OOC which source did you get this from?